n3fit/src/n3fit/io/writer.py

```diff
@@ -81,18 +71,12 @@ def write_data(self, replica_path_set, fitname, tr_chi2, vl_chi2, true_chi2):
             replica_path_set,
             fitname,
             self.q2,
-            replica_status.best_epoch,
-            vl_chi2,
-            tr_chi2,
-            true_chi2,
-            arc_lengths,
-            integrability_numbers,
-            allchi2_lines,
-            preproc_lines,
-            replica_status.positivity_status,
-            self.timings,
         )
 
+        # write the log file for the chi2
+        chi2_log = "\n".join(self.stopping_object.chi2exps_str())
+        (replica_path_set / "chi2exps.log").write_text(chi2_log)
```

Comment:
Yes, that and this is never read by vp.

Reply:
Ok. Though maybe that could be written to a json/yaml as well, to make life easier if we ever want to use it in vp?

Reply:
I'd rather just remove it at that point and maybe add an option to store _all_ chi2 (not only every 100 epochs).
I don't know, have you ever really found this log to be useful?

Reply:
On occasion I've found it useful to scroll through (a while after doing the fit, so the log was no longer available) so I'd rather not get rid of it entirely. Just wondering if it's worth it to make it a yaml since were touching this now anyway. In that case by the time (if ever) someone implements a reader most of the recent fits will be compatible.

Don't really have strong feelings about this though, just a thought.